server/shadow/shadow_capture.c

```diff
@@ -87,13 +87,12 @@ int shadow_capture_compare(BYTE* pData1, UINT32 nStep1, UINT32 nWidth, UINT32 nH
 	UINT32 nrow, ncol;
 	UINT32 l, t, r, b;
 	BYTE *p1, *p2;
-	BOOL rows[1024];
+	BOOL rows[1024] = { 0xff };
```

Comment:
Same here. `{ 0xFF }` sets the first element to `0xFF` and the rest (which is not specified) is automatically set to `0`, while before everything was set to `0xFF`.